mapiproxy/libmapistore/mapistore_python.c

```diff
@@ -2708,8 +2757,6 @@ static enum mapistore_error mapistore_python_message_set_read_flag(void *message
 		return MAPISTORE_ERR_CONTEXT_FAILED;
 	}
 
-	/* TODO Send Receipt */
```

Comment:
Good point, although at the moment we rely on the backend to return a mapistore error code. Since this happens in several methods, this change will affect the whole file, and we'll leave that for a future PR.